Makefile

```diff
@@ -473,11 +473,13 @@ CFLAGS += -I$(TVM_PATH)/include -DMXNET_USE_TVM_OP=1
 LDFLAGS += -L$(ROOTDIR)/lib -ltvm_runtime -Wl,-rpath,'$${ORIGIN}'
 
 TVM_USE_CUDA := OFF
+TVM_OP_CUDA_ARCH := NONE
```

Comment:
Any particular reason you are introducing a second set instead of using the arch set variable we already have? In which use case would these two differ?

Reply:
Reverted. Thanks for the suggestion.